<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="testM.TestStringVerification-resultado_Média" tests="3" file="testM.py" time="0.006" timestamp="2024-09-25T10:51:34" failures="3" errors="0" skipped="0">
	<testcase classname="testM.TestStringVerification" name="test_verificar_string_1" time="0.005" timestamp="2024-09-25T10:51:34" file="/home/fernando/Área de Trabalho/Projeto/Repositorio_IC/IC/Variáveis e estrutura sequencial/Média ponderada/Testes/testM.py" line="40">
		<failure type="AssertionError" message="False is not true"><![CDATA[Traceback (most recent call last):
  File "/home/fernando/Área de Trabalho/Projeto/Repositorio_IC/IC/Variáveis e estrutura sequencial/Média ponderada/Testes/testM.py", line 42, in test_verificar_string_1
    self.assertTrue(verificar_string("6.7", valores_entrada,self.file))
AssertionError: False is not true
]]></failure>
	</testcase>
	<testcase classname="testM.TestStringVerification" name="test_verificar_string_2" time="0.000" timestamp="2024-09-25T10:51:34" file="/home/fernando/Área de Trabalho/Projeto/Repositorio_IC/IC/Variáveis e estrutura sequencial/Média ponderada/Testes/testM.py" line="44">
		<failure type="AssertionError" message="False is not true"><![CDATA[Traceback (most recent call last):
  File "/home/fernando/Área de Trabalho/Projeto/Repositorio_IC/IC/Variáveis e estrutura sequencial/Média ponderada/Testes/testM.py", line 46, in test_verificar_string_2
    self.assertTrue(verificar_string("30.0", valores_entrada,self.file))
AssertionError: False is not true
]]></failure>
	</testcase>
	<testcase classname="testM.TestStringVerification" name="test_verificar_string_3" time="0.000" timestamp="2024-09-25T10:51:34" file="/home/fernando/Área de Trabalho/Projeto/Repositorio_IC/IC/Variáveis e estrutura sequencial/Média ponderada/Testes/testM.py" line="48">
		<failure type="AssertionError" message="False is not true"><![CDATA[Traceback (most recent call last):
  File "/home/fernando/Área de Trabalho/Projeto/Repositorio_IC/IC/Variáveis e estrutura sequencial/Média ponderada/Testes/testM.py", line 50, in test_verificar_string_3
    self.assertTrue(verificar_string("7.8", valores_entrada,self.file))
AssertionError: False is not true
]]></failure>
	</testcase>
</testsuite>
